packages/e2e-tests/next-app-dynamic-routes/pages/ssg-page.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { NextPageContext } from "next";
+import { GetStaticPropsContext } from "next";
```

Comment:
These import fixes are just [Boy Scout](https://deviq.com/boy-scout-rule/) changes after I noticed the type was incorrect in the test I was altering

Reply:
Thanks for fixing the types!